misc/completion/qutebrowser.zsh

```diff
@@ -0,0 +1,34 @@
+#compdef qutebrowser
+
+local ret=1
+
+_arguments -s : \
+  '(- *)'{-h,--help}'[show the help message]' \
+  '(- *)'{-V,--version}'[show version information]' \
+  '(-h --help -V --version --basedir)--basedir[base directory for all storage.]:storage base directory:_directories' \
+  '(-h --help -V --version)'-{s,-set}'[set a temporary setting for this session]:option:' \
+  '(-h --help -V --version -r --restore)'-{r,-restore}'[restore a named session]:session name:' \
+  '(-h --help -V --version -R --override-restore)'-{R,-override-restore}'[do not restore a session]' \
+  '(-h --help -V --version --target)--target[how to open URLs in running instance]:open URLs in:(auto tab tab-bg tab-silent tab-bg-silent window)' \
+  '(-h --help -V --version --backend)--backend[which backend to use]:backend:(webkit webengine)' \
+  '(-h --help -V --version --enable-webengine-inspector)--enable-webengine-inspector[enable web inspector for QtWebEngine]' \
+  '(-h --help -V --version -l --loglevel)'-{l,-loglevel}'[set loglevel]:log level:(critical error warning info debug vdebug)' \
+  '(-h --help -V --version --logfilter)--logfilter[what to log to stdout]:log items:' \
+  '(-h --help -V --version --loglines)--loglines[log lines to keep in RAM]:lines:_integer' \
```

Comment:
`--loglevel`, `--logfilter` and `--loglines` probably shouldn't be in here, they're not really relevant to users. Not sure about `--debug`, I guess it's okay to keep that.